content/tutorial/react/index.md

```diff
@@ -33,16 +33,56 @@ In your LoginRadius Dashboard, navigate to **<a href="https://dashboard.loginrad
 
 ![alt_text](../../assets/blog-common/api-credentials.png "image_tooltip")
 
+## Whitelist Your Domain
+
+For security reasons, LoginRadius processes the API calls that are received from the whitelisted domains. Local domains (http://localhost and http://127.0.0.1) are whitelisted by default.
+
+To whitelist your domain, in your LoginRadius Dashboard, navigate to **<a href="https://dashboard.loginradius.com/configuration" target="_blank">Configuration > Whitelist Your Domain</a>** and add your domain name:
+
+![alt_text](../../assets/blog-common/domain-whitelisting.png "image_tooltip")
+
 ## Setup React JS
 
-This example uses a sample app based on the Create React App (CRA) boilerplate. For directions on how to Create React App, you can reference <a href="https://reactjs.org/docs/create-a-new-react-app.html" target="_blank">here.</a>
+This example uses a sample app based on the Create React App (CRA) boilerplate. For directions on how to Create React App, you can reference <a href="https://reactjs.org/docs/create-a-new-react-app.html" target="_blank">here.</a> 
 
 Once the CRA boilerplate is set up, follow these steps:
 
 - Navigate to the project root and install `react-router-dom`:
 
   `npm install react-router-dom`
 
+- We will also be using the [LoginRadius React SDK](https://github.com/LoginRadius/loginradius-react) to implement different functionalities. 
```

Comment:
This line should come under the header **Install LoginRadius React SDK**. And explain in short what does login radius react SDK does. Something like this 
```
The LoginRadius React SDK uses [React Context](https://reactjs.org/docs/context.html) to manage the authentication state of your users. One way to integrate LoginRadius with your React app is to wrap your root component with an `LRAuthProvider` that you can import from the SDK.
```